test/fastboot-test.js

```diff
@@ -81,7 +81,7 @@ describe("FastBoot", function() {
 
   it("can forcefully destroy the app instance using destroyAppInstanceInMs", function() {
     var fastboot = new FastBoot({
-      distPath: fixture('basic-app')
+      distPath: fixture('slow-app')
```

Comment:
Is this from your other PR of where you were refactoring the destoryAppInstance stuff?